### PR TITLE
feature/JP-64-page-transitions

### PR DESCRIPTION
**Pull request by EinSev**

### CODE REVIEW

File: frontend/pages/journey/new.vue [hunk 1]
```diff
@@ -5,16 +5,23 @@ import Toast from "primevue/toast";
...
 const journeyInvite = uuidv4();
 
+const title = t.value("title.journey.create");
+useHead({
+  title: `${title} | JourneyPlanner`,
+})
+
+definePageMeta({
+  //TODO
+  //middleware: ["sanctum:auth"],
```
Comment (Mathematti) [inline frontend/pages/journey/new.vue]: TODO

File: frontend/layouts/default.vue [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
+        <footer
+            class="text-footer text-xs md:text-sm font-nunito pb-3 flex items-center justify-center overflow-hidden">
+            <a href="mailto:contact@journeyplanner.io"
+                class="hover:font-bold hover:text-link">contact@journeyplanner.io</a>
+            <span class="mx-2"> | </span>
+            <a href="https://github.com/JourneyPlanner/JourneyPlanner" target="_blank"
+                class="hover:font-bold hover:text-link">GitHub</a>
+        </footer>
+    </div>
+</template>
```
Comment (Mathematti) [inline frontend/layouts/default.vue]: Missing empty line at the end of the file

**Review by Ixpiria — APPROVED** [review]

**Review by stefll — CHANGES_REQUESTED** [review]
Design Review: All good

**Review by stefll — APPROVED** [review]
Design Review: All Good

**Pull request merged**